helix-term/src/commands.rs

```diff
@@ -1000,31 +1000,70 @@ fn goto_file_vsplit(cx: &mut Context) {
     goto_file_impl(cx, Action::VerticalSplit);
 }
 
+fn resolve_selected_path(
+    selection_text: String,
+    doc_path_parent: Option<&Path>,
+) -> Option<PathBuf> {
+    let trimmed = selection_text.trim();
+    if trimmed.is_empty() {
+        return None;
+    }
```

Comment:
Why not just allow it to continue so it can open up current directly?

Reply:
This is just a reproduction of the current logic. I could play around with this, as I am not sure, what will happen, if I use the empty string as file or directory name in the respective functions.

Reply:
After thinking about this a bit it is my opinion, it is good as it is.

To open the file picker, we have `<Space>+f`.

And `gf` on a selection containing whitespace opening the file picker for the current document directory is counterintuitive in my opinion. Thats what `gf` should do on `./` or some such.

Reply:
Sorry, I don't understand, what you want to express with the eyes emoji? Am I too stubborn?

Reply:
Ah, it means I looked at your comment, but I don't have an opinion.

> Am I too stubborn?

No

Reply:
I stubbornly agree with @typetetris :) I am a new Helix user and it really feels like a bug to me, that the relative path is not resolved relative to the current document.

Reply:
I guess should be fine for now, I was probably confused back then.